File: services/device/src/main/java/com/example/iter/device/dto/response/AdminEquipmentSummaryResponse.java
package com.example.iter.device.dto.response;

import com.example.iter.auth.api.UserSummary;
import com.example.iter.device.domain.entity.EquipmentCategory;
import com.example.iter.device.domain.entity.EquipmentStatus;
import com.example.iter.device.domain.entity.ProductConditionType;

import java.math.BigDecimal;
import java.time.LocalDate;
import java.time.LocalDateTime;
import java.util.List;

public record AdminEquipmentSummaryResponse(
        Long equipmentId,
        String name,
        EquipmentCategory category,
        BigDecimal dailyPrice,
        EquipmentStatus status,
        UserSummary owner,
        String thumbnailUrl,
        LocalDateTime createdAt
) {

}

File: services/device/src/main/kotlin/com/example/iter/device/dto/response/AdminEquipmentSummaryResponse.kt
package com.example.iter.device.dto.response

import com.example.iter.auth.api.UserSummary
import com.example.iter.device.domain.entity.EquipmentCategory
import com.example.iter.device.domain.entity.EquipmentStatus

import java.math.BigDecimal
import java.time.LocalDateTime

data class AdminEquipmentSummaryResponse(
    val equipmentId: Long,
    val name: String,
    val category: EquipmentCategory,
    val dailyPrice: BigDecimal,
    val status: EquipmentStatus,
    val owner: UserSummary,
    val thumbnailUrl: String?,
    val createdAt: LocalDateTime?,
)
